keras_core/backend/tensorflow/distribution_lib.py

```diff
@@ -0,0 +1,63 @@
+"""!!!DO NOT USE!!!
+
+Distribution related class for Tensorflow backend.
+
+This is just a prototype and we might want to unify it
+with other backends in the future.
+"""
+import tensorflow as tf
+from tensorflow.experimental import dtensor
+
+
+def list_devices(device_type=None):
+    """Return all the available devices based on the device type.
+    
+    Note that this should return the global devices in a distributed setting.
+    
+    Args:
+        device_type: string of `"cpu"`, `"gpu"` or `"tpu"`. Default to `gpu` or
+        `tpu` if available when device_type is not provided. Otherwise will
+        return the `cpu` devices.
+
+    Return:
+        List of devices that are available for distribute computation.
+    """
+    device_type = (
+        device_type.lower() if device_type else dtensor.preferred_device_type()
+    )
+    return dtensor.local_devices(device_type=device_type)
```

Comment:
This seems only return the local devices, which is different from the docstring.

Reply:
Seems that this is not addressed, did I miss anything?